app/Http/Controllers/DashboardController.php

```diff
@@ -36,4 +45,46 @@ private function getTopLoadouts()
 
         return $loadouts;
     }
+
+    private function getRecentTopLoadouts()
+    {
+        $loadouts = collect();
+        $characterIds = Character::pluck('id')->sortBy('id');
+
+        foreach ($characterIds as $characterId) {
+            $characterLoadouts = Loadout::where([
+                ['character_id', $characterId],
+                ['created_at', '>', Carbon::now()->subDays(14)], // Loadouts created in the past 2 weeks
+            ])
+                ->withCount('votes')
+                ->with('character', 'creator', 'mods.gun')
+                ->orderBy('votes_count', 'desc')
+                ->take(3)
+                ->get();
+            $loadouts->push($characterLoadouts);
+        }
+
+        return $loadouts;
+    }
+
+    private function getLatestLoadouts()
+    {
+        $loadouts = collect();
+        $characterIds = Character::pluck('id')->sortBy('id');
+
+        foreach ($characterIds as $characterId) {
+            $characterLoadouts = Loadout::where([
+                ['character_id', $characterId],
+                ['created_at', '>', Carbon::now()->subDays(14)], // Loadouts created in the past 2 weeks
+            ])
+                ->withCount('votes')
+                ->with('character', 'creator', 'mods.gun')
+                ->orderBy('created_at', 'desc')
```

Comment:
You don't have to change it, more of a TIL maybe for ya: you can use the `latest()` function for this as well.

Reply:
Did this locally, will push up shortly. Good note thanks 👍